### PR TITLE
Icon() expected Fl_RGB_Image and received Pixmap. FLTK icons work now.

### PR DESCRIPTION
**Pull request by thenorili**

### CODE REVIEW

**Comment by thenorili** [comment]
Probably the cause of #https://github.com/zynaddsubfx/zyn-fusion-build/issues/30

File: src/UI/MasterUI.fl [hunk 1]
```diff
@@ -87,6 +87,22 @@ decl {\#if !defined(PLUGINVERSION) && HAS_X11
...
 } 
 
+decl {\#if !defined(PLUGINVERSION) && HAS_X11
+\#include <FL/Fl_RGB_Image.H>
+\#endif} {private local
+} 
+
+decl {\#if !defined(PLUGINVERSION) && HAS_X11
+\#include <FL/FL.H>
+\#include "Enumerations.H"
```
Comment (fundamental) [inline src/UI/MasterUI.fl]: That should be #include <FL/Enumerations.H>, right?

Comment (thenorili) [reply src/UI/MasterUI.fl]: Enumerations.H is a file, not a class. In the header files and example code I read, it was included in this manner.

Comment (thenorili) [reply src/UI/MasterUI.fl]: I found a piece of example code (/test/sudoku.cxx) that includes it in this manner, so I'll make the change.

**Comment by thenorili** [comment]
I'm going to review it further, but it may be true that the XCreatePixmapFromData call was the only function call to X11/xpm.H and it can be removed from the codebase.

**Comment by fundamental** [comment]
Thanks for the update :+1:

**Pull request merged**